source/guide_castopod.rst

```diff
@@ -0,0 +1,193 @@
+
+Prerequisites
+=============
+
+We're using :manual:`PHP <lang-php>` in the version 8.1:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version use php 8.1
+ Selected PHP version 8.1
+ The new configuration is adapted immediately. Patch updates will be applied automatically.
+
+ [isabell@stardust ~]$
+
+.. include:: includes/my-print-defaults.rst
+
+Setup a new MySQL database for Castopod:
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ mysql -e "CREATE DATABASE ${USER}_castopod"
+  [isabell@stardust ~]$
+
+Your castopod URL needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+.. note::
+  Castopod should be installed in a domain of its own (or in a subdomain),
+  not in a subdirectory of a domain.
+
+Installation
+============
+
+First, go one level above your :manual:`DocumentRoot <web-documentroot>`
+and download the latest version. Make sure, it is a Castopod Package and not
+the source code. At time of this writing this is v1.0.0-beta.14.
+
+TODO: the permalink to latest version requires login:
+https://docs.gitlab.com/ee/user/project/releases/#permanent-link-to-latest-release
+
+Download the source
+-------------------
+
+::
+
+  [isabell@stardust ~]$ cd /var/www/virtual/$USER
+  [isabell@stardust isabell]$ wget https://code.castopod.org/adaures/castopod/uploads/48710de086e7b4dfa23f894e81e15365/castopod-1.0.0-beta.14.tar.gz
+  [isabell@stardust isabell]$
+
+Unpack and symlink
+------------------
+
+Unpack the tarball and remove the downloaded file.
+
+::
+
+  [isabell@stardust ~]$ cd /var/www/virtual/$USER
+  [isabell@stardust isabell]$ tar --extract --verbose --ungzip --file castopod-1.0.0-beta.14.tar.gz
+  [isabell@stardust isabell]$ rm castopod-1.0.0-beta.14.tar.gz
+  [isabell@stardust isabell]$
+
+Remove your empty DocumentRoot and create a new symbolic link
+to the castopod directory:
+
+::
+
+  [isabell@stardust ~]$ cd /var/www/virtual/$USER
+  [isabell@stardust isabell]$ rm --force html/nocontent.html; rmdir html
+  [isabell@stardust isabell]$ ln --symbolic /var/www/virtual/$USER/castopod/public html
+  [isabell@stardust isabell]$
+
+Update .htaccess
+----------------
+
+Open the .htaccess file in html and remove unsupported lines: All and FollowSymLinks.
+
+TODO: Codeblock needed here?
+
+Finishing installation
+======================
+
+Before finishing the installation via wizard, set up some cronjobs for
+various background processes:
+
+For social features to work properly, this task is used to broadcast social activities
+to your followers on the fediverse:
+
+.. code-block:: none
+
+   */10 * * * * cd /var/www/virtual/$USER/html && php index.php scheduled-activities
+
+For having your episodes be broadcasted on open hubs upon publication using WebSub:
+
+.. code-block:: none
+
+   */10 * * * * cd /var/www/virtual/$USER/html && php index.php scheduled-websub-publish
+
+Point your browser to your uberspace URL or domain.
+You will see an error page. Add ``cp-install`` to the URL and finish the
```

Comment:
just give the domain https://isabell.uber.space/cp-install and advise users to change it accordingly